docs/azure/sdk/authentication/index.md

```diff
@@ -41,24 +35,45 @@ Token-based authentication offers the following advantages over authenticating w
 
 Use of connection strings should be limited to initial proof-of-concept apps or development prototypes that don't access production or sensitive data. Otherwise, the token-based authentication classes available in the Azure Identity library should always be preferred when authenticating to Azure resources.
 
-## Authentication in server environments
+## Authentication for Azure-hosted apps
+
+When your app is hosted on Azure, it can leverage Azure's managed identities to authenticate to Azure resources without needing to manage any credentials. There are two types of managed identities: user-assigned and system-assigned. Both types provide a secure way to authenticate to Azure services.
```

Comment:
```suggestion
When your app is hosted on Azure, it can use managed identities to authenticate to Azure resources without needing to manage any credentials. There are two types of managed identities: user-assigned and system-assigned. Both types provide a secure way to authenticate to Azure services.
```